Sources/PackageGraph/PackageGraphLoader.swift

```diff
@@ -252,8 +252,10 @@ private func createResolvedPackages(
         // The diagnostics location for this package.
         let diagnosticLocation = { PackageLocation.Local(name: package.name, packagePath: package.path) }
 
-        // Get all the system module dependencies in this package.
-        let systemModulesDeps = packageBuilder.dependencies
+        // Implicitly add system targets from dependencies, if their
+        // manifest is less than version 4.1.0.
```

Comment:
I think we need to remember if the system target was implicit or explicit, and only do this for implicit ones (ones defined by system module packages).

Reply:
:(